### DISCUSSION TITLE
lightning_threshold does not work even if i set it to minimum

### DISCUSSION THREAD

**Discussion by akcasoy** [blakeblackshear/frigate#14578] [Ask A Question] [answered]
I am trying to fine-tune my configuration for my daughters room and wanted to try **lightning_threshold** in motion, in order not to record the times where we nights just turn on the IR of the camera shortly, to see if everything is OK.

The default value is 0.8. And this is stated in documentation:

```
  # Increasing this value will make motion detection more likely to consider lightning or ir mode changes as valid motion.
  # Decreasing this value will make motion detection more likely to ignore large amounts of motion such as a person approaching
```

So since my first try with the default value did not work (it still recorded), so i have **decreased** the value (since i do not want that the frigate considers that time as a valid motion). At the moment the value is at 0.3 (minimum, <0.3 gives you compile error), and it does not work.

Do i miss here something? When i open the room light or IR of the camera at night, there is a big sudden change in the footage (as expected), but the frigate is still recording this as motion.

Thank you in advance for your help.

**Comment by NickM-27** [comment]
Lightning threshold does not stop motion from being detected entirely. Instead, it stops additional motion detection from running once an initial amount of motion is detected above the threshold.

**Reply by akcasoy** [reply]
In documentation it is written "_Large changes in motion like PTZ moves and camera switches between Color and IR mode **should result in no motion detection**_" ?

**Reply by NickM-27** [reply]
I'll update that, the feature is designed to reduce false positive object detections and motion usage during periods of high motion. It will not stop recordings from being saved as users still expect their ptz camera to record while it is moving, for example.

**Reply by akcasoy** [reply]
> "It will not stop recordings from being saved as users still expect"

But you do not take into account (like the documentation mostly also does) the people, who just want to record motion without object detection. They would NOT want "their ptz camera to record while it is moving". Since the software CAN distinguish if that is really a legit motion to record, or a high-motion (IR Light going on/off).

**Reply by NickM-27** [reply, marked as answer]
The consequence of not recording during this is the potential of missing something important like a ptz camera following an object using auto tracking or seeing something while moving. 

The consequence of keeping this motion recording is potentially a couple recordings segments that take up a few megabytes and something that takes almost no time at all to review in the timeline in the UI. 

This is working as intended. I have put up a PR to add a note to the motion docs about this behavior.

**Reply by akcasoy** [reply]
> The consequence of not recording during this is the potential of missing something important like a ptz camera following an object using auto tracking or seeing something while moving.

ok you are right.. Probably there is not a easy solution for everybody.
Thank you again Nick.
